benchmarks/7.0.0/v3/summary-graph.md

```diff
@@ -0,0 +1,306 @@
+| Concurrent Users                  | The number of users accessing the application at the same time.                                                   | 50, 100, 150, 300, 500, 1000, 1500, 2000, 2500, 3000            |
+| IS Instance Type 2 Cores          | The AWS EC2 instance type used to run the Identity Server.                                                        | [**c5.large**](https://aws.amazon.com/ec2/instance-types/)      |
+| IS Instance Type 4 Cores          | The AWS EC2 instance type used to run the Identity Server.                                                        | [**c5.xlarge**](https://aws.amazon.com/ec2/instance-types/)     |
+| RDS Instance Type                 | The AWS RDS instance type used to run the Identity Server.                                                        | [**db.m4.2xlarge**](https://aws.amazon.com/rds/instance-types/) |
+| RDS Instance max_connections      | The AWS RDS max_connections metric monitors the set maximum number of (allowed) simultaneous client connections.  | 2500                                                            |
+| JDK version                       | The JDK version used to run the Identity Server.                                                                  | JDK 11.0.15.1                                                   |
+
+Product Configurations: deployment.toml
+
+```
+[user_store.properties]
+CaseInsensitiveUsername = false
+SCIMEnabled=true
+IsBulkImportSupported=false
+
+[database.identity_db.pool_options]
+maxActive = "400"
+
+[database.shared_db.pool_options]
+maxActive = "400"
+
+[database.user.pool_options]
+maxActive = "400"
+
+[transport.https]
+maxThreads = "400"
+acceptCount = "400"
+```
+
+The following is the summary of performance test results collected for the measurement period.
+
+### 1. Client Credentials Grant Type
+
+#### Obtain an access token using the OAuth 2.0 client credential grant type.
+
+Performance Comparison of Different Node Configurations with 95th Percentile of Response Time (ms)
+
+| Concurrent Users | Active-Passive 4 Core | Two Node 2 Core | Two Node 4 Core | Three Node 4 Core | Four Node 4 Core |
+|------------------|-----------------------|-----------------|-----------------|-------------------|------------------|
+|   50   |   203  |   329  |   82   |   70   |   54   |
+|  100   |   481  |   771  |   285  |   145  |   108  |
+|  150   |   783  |  1295  |   495  |   220  |   163  |
+|  300   |  1863  |  2319  |  1191  |   751  |   423  |
+|  500   |  2191  |  3359  |  1775  |  1391  |   891  |
+|  1000  |  3535  |  6047  |  2719  |  2255  |  2239  |
+|  1500  |  4831  |  8767  |  4127  |  4127  |  4287  |
+|  2000  |  6079  | 10367  |  4831  |  4991  |  4703  |
+|  2500  |  7359  | 11135  |  5919  |  6655  |  5439  |
+|  3000  |  8831  | 17791  |  5279  |  7551  |  7167  |
+
+<ins> Concurrency: 50 - 500 </ins>
+
+![image info](graphs/Client_Credentials_Grant_Type/50_500_lines.png)
```

Comment:
Images are not visualised in the summary file.
Can you please check whether the path is correct?